bb_tracking/models.py

```diff
@@ -76,30 +76,56 @@ def to_dict(self):
                     axis_normalizers=(self.axis_normalizers_min_, self.axis_normalizers_max_),
                     diff_normalizer=(self.diff_normalizer_min_, self.diff_normalizer_max_)
                    )
-    
+
     @classmethod
+    # def load(cls, path):
```

Comment:
Commented out code should be removed (the history will always be available anyway)